gnd/src/main/java/com/google/android/gnd/ui/map/GroundCameraPosition.java

```diff
@@ -0,0 +1,27 @@
+package com.google.android.gnd.ui.map;
+
+import com.google.android.gnd.model.feature.Point;
+
+public class GroundCameraPosition {
+
+  private final Point target;
+  private final Float zoomLevel;
+
+  public GroundCameraPosition(Point target, Float zoomLevel) {
```

Comment:
We haven't been prefixing classes with `Ground` since the package name sufficiently disambiguates. Could we use `CameraPosition` instead?

Reply:
Sure. There's already a `CameraPosition` class in `com.google.android.gms.maps.model` so I added the `Ground` prefix to our class to avoid having to qualify usage of both classes in `GoogleMapsMapAdapter`, however, it's only one line so actually `CameraPosition` makes much more sense.